src/Microsoft.ApplicationInsights/Extensibility/Implementation/Tracing/DiagnosticsTelemetryModule.cs

```diff
@@ -42,6 +46,48 @@ public DiagnosticsTelemetryModule()
             this.Dispose(false);
         }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether or not the Health Heartbeat feature is disabled.
+        /// </summary>
+        public bool IsHeartbeatEnabled
```

Comment:
is this tested?

Reply:
Yes, more fully now than before. Thanks for calling it out.